### DISCUSSION TITLE
how to set local module path(yarn berry pnp mode) in Yarn 1.xx project?

### DISCUSSION THREAD

**Discussion by jhlee910609** [yarnpkg/berry#3379] [Q&A]
- I'm migrating Yarn 1 to Yarn Berry.
- I want to use node modules in the Yarn Berry project.  But I have a problem using local modules...
- plz, help me. Here is my `package.json`.
```
"dependencies": {
    "@custom_lib": "file:../..",
    "@types/react": "file:../../node_modules/react",
    "@types/react-dom": "file:../../node_modules/react-dom",
    "next": "11.1.0",
    "react": "file:../../node_modules/react",
    "react-dom": "file:../../node_modules/react-dom"
  },
```

**Comment by larixer** [comment]
Your intent is unclear here. Please describe what you are trying to achieve?

**Reply by jhlee910609** [reply]
@larixer `@custom_lib` is used by `yarn berry pnp mode`. So, `@custom_lib` hasn't `node_modules`. but, this package is not used by `yarn berry, just yarn 1.x`. i wanna local module `@custom_lib` link to yarn 1.x package.
